mmcv/ops/csrc/common/cuda/bezier_align_cuda_kernel.cuh

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) OpenMMLab. All rights reserved
+#ifndef BEZIER_ALIGN_CUDA_KERNEL_CUH
```

Comment:
As the code is modified from other repo. We should add some comments here, for example:
```dotnetcli
// modified from
// https://the.link.of.the.code.in.origin.repo
```